server.js

```diff
@@ -367,6 +382,13 @@ app.get("/username/collectionValue", (req, res) => {
       res.status(500).send(err);
     });
 });
+//   //   });
+//   // });
+// });
+app.use(
+  "/resources/mario.jpg",
+  express.static(path.join(__dirname, "resources", "mario.jpg"))
+);
```

Comment:
Let's remove unused code before merging.

Reply:
This is used when signing in as the default avatar assignment to the users